src/core/ext/transport/chttp2/transport/internal.h

```diff
@@ -270,7 +270,6 @@ struct grpc_chttp2_transport {
   grpc_slice_buffer outbuf;
```

Comment:
Strongly suggest splitting out:
```
struct grpc_chttp2_stream_flow_control
struct grpc_chttp2_transport_flow_control
```
as separate types, and containing them in grpc_chttp2_stream, grpc_chttp2_transport as fields (`grpc_chttp2_XXX_flow_control flow_control` for instance).

Then the only interface between flow_control.c and the rest of chttp2 should be via these structs

This will:
1. ensure honesty in layering moving forward (by preventing additional chttp2 details getting placed into flow_control.c)
2. make any access into the flow control layer from chttp2 easily greppable